### DISCUSSION TITLE
[Question] Does WSL support running init script?

### DISCUSSION THREAD

**Discussion by ThomsonTan** [microsoft/WSL#6092] [Q&A] [answered]
Seems WSL/WSL2 runs its own `init` as `/init` and not the one from distro. Does it support running init script like `etc/inittab` or is there any way to do that?

**Comment by craigloewen-msft** [comment, marked as answer]
In general WSL has its own custom init process. In general we don't yet have strong support for init scripts beyond scheduling them using WIndows tasks, or putting lines inside of your ~/.bashrc file.
